tests/script/test_script.py

```diff
@@ -511,3 +513,20 @@ def testCheckMethodCalls(SAMPLE_PATH_14d9f) -> None:
     assert checkMethodCalls("14d9f1a92dd984d6040cc41ed06e273e.apk", targetMethod, checkMethods) is True
 
 
+def testFindMethodImpls(SAMPLE_PATH_pivaa) -> None:
+    abstractMethod = [
+        "Ljavax/net/ssl/HostnameVerifier;",
+        "verify",
+        "(Ljava/lang/String; Ljavax/net/ssl/SSLSession;)Z"
+    ]
+    methodImpls = findMethodImpls(SAMPLE_PATH_pivaa, abstractMethod)
+    assert len(methodImpls) == 1
```

Comment:
Could you update the unit test to check the class name, method name, and descriptor of the implementation?